src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -54,6 +56,14 @@ public void update(StandaloneConfig standaloneConfig, List<String> ignoreGlobLis
             List<String> aliases = new ArrayList<>(author.getAuthorAliases());
             List<String> emails = new ArrayList<>(author.getEmails());
             aliases.add(author.getGitId());
+            allAliases.addAll(aliases);
+
+            HashSet<String> uniqueAliasesSet = new HashSet<>(allAliases);
+            boolean hasDuplicateAliases = uniqueAliasesSet.size() != allAliases.size();
```

Comment:
why need allAliases? can just compare size of uniqueAliases with aliases

Reply:
aliases is just keeping a track of the aliases of a specific author and not all the aliases of all authors. This is why need allAliases.

Reply:
why not do this outside of the for loop then?

Reply:
Also can. Actually will be better I agree.